<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>master</class>
 <widget class="QMainWindow" name="master">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChmcuInit</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1" rowspan="2">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCommandLinkButton" name="start_project">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">.QCommandLinkButton
{

    color:#ffffff;
	background-color: rgb(123, 199, 233);
    border-style:outset; 
    border-width:2px;
    border-color:rgb(123, 199, 233);
    border-radius:10px;
    font:bold 14px; 
    font-family: Segoe UI;
    min-width:100px;
    min-height:20px;
    padding:4px;

}</string>
          </property>
          <property name="text">
           <string>开始项目</string>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/logo/res/logo/start project.png</normaloff>:/logo/res/logo/start project.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QTreeWidget" name="mcu_info">
          <column>
           <property name="text">
            <string>芯片主型号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>型号参考</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>封装</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Flash</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>RAM</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>IO</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>频率</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>HC32F002</string>
           </property>
           <property name="text">
            <string>HC32F002D4PB</string>
           </property>
           <property name="text">
            <string>TSSOP24</string>
           </property>
           <property name="text">
            <string>18KBytes</string>
           </property>
           <property name="text">
            <string>2KByes</string>
           </property>
           <property name="text">
            <string>22</string>
           </property>
           <property name="text">
            <string>48Mhz</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2">
         <widget class="QTreeWidget" name="mcu_choose">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>内容</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>华大MCU</string>
           </property>
           <item>
            <property name="text">
             <string>HC32F0</string>
            </property>
            <item>
             <property name="text">
              <string>H32F002</string>
             </property>
            </item>
           </item>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" rowspan="2">
         <widget class="QTabWidget" name="tabWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>引脚</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="window_pin" name="pin_tree">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="midLineWidth">
               <number>0</number>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>外设</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QTreeWidget" name="treeWidget_4">
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGraphicsView" name="mcu_view"/>
        </item>
        <item row="0" column="3" rowspan="2">
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QTableWidget" name="pin_table">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="4" rowspan="2">
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>寄存器</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QTreeWidget" name="treeWidget_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string>源代码</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QTextBrowser" name="textBrowser"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>973</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionddd"/>
    <addaction name="actionddd_2"/>
    <addaction name="separator"/>
    <addaction name="actiond"/>
    <addaction name="actiond_2"/>
    <addaction name="separator"/>
    <addaction name="actiond_3"/>
    <addaction name="separator"/>
    <addaction name="actiond_4"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionddd">
   <property name="text">
    <string>新建项目</string>
   </property>
  </action>
  <action name="actionddd_2">
   <property name="text">
    <string>打开项目</string>
   </property>
  </action>
  <action name="actiond">
   <property name="text">
    <string>保存项目</string>
   </property>
  </action>
  <action name="actiond_2">
   <property name="text">
    <string>另存为项目</string>
   </property>
  </action>
  <action name="actiond_3">
   <property name="text">
    <string>关闭项目</string>
   </property>
  </action>
  <action name="actiond_4">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actiond_5">
   <property name="text">
    <string>d</string>
   </property>
  </action>
  <action name="actiond_6">
   <property name="text">
    <string>d</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>window_pin</class>
   <extends>QTreeWidget</extends>
   <header>window_pin.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
